source/whats-new.txt

```diff
@@ -40,37 +40,65 @@ Learn what's new in:
 What's New in 5.0
 -----------------
 
-.. warning:: Breaking Changes in v5.0
+.. warning:: Breaking changes in this release
```

Comment:
I see that the warning about deprecations in 4.11 (the latest release at the moment) uses the "Deprecations in **this release**" wording. This suggests that the wording was changed since release 4.8, where the wording is "Breaking Changes in **v4.8**". @rachel-mack Are you sure the old wording should be used again?